addons/jest/src/components/Message.tsx

```diff
@@ -0,0 +1,212 @@
+/* tslint:disable:object-literal-sort-keys */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { styled } from '@storybook/theming';
+import colors from '../colors';
```

Comment:
I think these colours should come from the theme if possible

Reply:
Hmm, I don't find similar colors in `lib/theming/src/base.ts`